frontend/file/build.gradle.kts

```diff
@@ -17,12 +17,14 @@ apply<GrammarKitPlugin>()
 dependencies {
   api(project(":qual"))
   implementation(project(":binding"))
+  implementation(project(":branchLayout:impl"))
```

Comment:
Don't depend on implementations in general, depend just on API.

See javadoc for `com.virtuslab.binding.RuntimeBinding#instantiateSoleImplementingClass` for the rationale